### PR TITLE
Ita translation

### PR DESCRIPTION
**Pull request by Giuzy**

### CODE REVIEW

**Comment by Giuzy** [comment]
Hi Toffer, I want to point out that I changed 6 tags because the text didn't appear on screen and the tag was there instead.
They are:
TXT_KEY_BONUS_ROMANI into TXT_KEY_BONUS_ROMANE (file Bonus_CIV4GameText)
TXT_KEY_PROMOTION_AN_DESC into TXT_KEY_PROMOTION_AN (file Global_CIV4GameText)
TXT_KEY_PROMOTION_ENKI_DESC into TXT_KEY_PROMOTION_ENKI (same as above)
TXT_KEY_PROMOTION_ENLIL_DESC into TXT_KEY_PROMOTION_ENLIL (same as above)
TXT_KEY_PROMOTION_INANNA_DESC into TXT_KEY_PROMOTION_INANNA (same as above)
TXT_KEY_PROMOTION_KI_DESC into TXT_KEY_PROMOTION_KI (same as above)
TXT_KEY_BUILDING_HUMAN_MOD_NON_SPATIAL into TXT_KEY_BONUS_HUMAN_MOD_NON_SPATIAL (file Buildings_CIV4GameText)

After the change, they all appear correctly on screen but please let me know in case this change was not meant to be done for whatever reason and I'll just change them back quickly.

For the rest, I changed editor and using Notepad++ now and as you said it's better than VS and it doesn't seem to leave any blanks when inserting new text. So I hope that this time there should be no problem for that, I checked before and after sending the pull request and it was all fine but please let me know if this is not the case.

Thanks have a nice evening! Ciao.

**Comment by Giuzy** [comment]
Hello, I'm progressing with the translation and I've logged already another good bunch of changes in Kraken (for sure more that 1k, may be 2k). Do you think I can add them here or better wait and do another request to not charge it too much (this one is already quite big as it has a spell check)? It's quite difficult to keep the changes to just a few files as the texts are quite scattered around and I'm trying to maintain some consistency.  Thank you, nice week-end

**Comment by Giuzy** [comment]
Hello, I'm progressing with the translation and I've logged already another good bunch of changes in Kraken (for sure more that 1k, may be 2k). Do you think I can add them here or better wait and do another request to not charge it too much (this one is already quite big as it has a spell check)? It's quite difficult to keep the changes to just a few files as the texts are quite scattered around and I'm trying to maintain some consistency. Thank you, nice week-end

File: Assets/XML/GameText/Bonus_CIV4GameText.xml [hunk 1]
```diff
@@ -7800,7 +7800,7 @@
 		<Russian>Ðèìëÿíå</Russian>
 	</TEXT>
 	<TEXT>
-		<Tag>TXT_KEY_BONUS_ROMANI</Tag>
+		<Tag>TXT_KEY_BONUS_ROMANE</Tag>
 		<English>Romanì</English>
 		<Russian>Öûãàíå</Russian>
```
Comment (Toffer90) [inline Assets/XML/GameText/Bonus_CIV4GameText.xml:7805]: Looks like you have a mistake here.

Comment (Giuzy) [reply Assets/XML/GameText/Bonus_CIV4GameText.xml:7805]: Hey Toffer, please read the comment above it's intentional, but if you prefer I'll change them back.

Comment (Giuzy) [reply Assets/XML/GameText/Bonus_CIV4GameText.xml:7805]: Please let me know if I should do something.

**Comment by Toffer90** [comment]
> Hi Toffer, I want to point out that I changed 6 tags because the text didn't appear on screen and the tag was there instead. They are: TXT_KEY_BONUS_ROMANI into TXT_KEY_BONUS_ROMANE (file Bonus_CIV4GameText) TXT_KEY_PROMOTION_AN_DESC into TXT_KEY_PROMOTION_AN (file Global_CIV4GameText) TXT_KEY_PROMOTION_ENKI_DESC into TXT_KEY_PROMOTION_ENKI (same as above) TXT_KEY_PROMOTION_ENLIL_DESC into TXT_KEY_PROMOTION_ENLIL (same as above) TXT_KEY_PROMOTION_INANNA_DESC into TXT_KEY_PROMOTION_INANNA (same as above) TXT_KEY_PROMOTION_KI_DESC into TXT_KEY_PROMOTION_KI (same as above) TXT_KEY_BUILDING_HUMAN_MOD_NON_SPATIAL into TXT_KEY_BONUS_HUMAN_MOD_NON_SPATIAL (file Buildings_CIV4GameText)
> 
> After the change, they all appear correctly on screen but please let me know in case this change was not meant to be done for whatever reason and I'll just change them back quickly.
> 
> For the rest, I changed editor and using Notepad++ now and as you said it's better than VS and it doesn't seem to leave any blanks when inserting new text. So I hope that this time there should be no problem for that, I checked before and after sending the pull request and it was all fine but please let me know if this is not the case.
> 
> Thanks have a nice evening! Ciao.

I see, yeah all the promotion tags you changed were bug fixes, but you need to revert the change to TXT_KEY_BUILDING_HUMAN_MOD_NON_SPATIAL as the change makes the building lose its name (just check the pedia index).

**Pull request merged**